### PR TITLE
Add scratch attribute to koji builds

### LINKED ISSUE #1338: Differentiate scratch and non-scratch builds in a database
* [ ] Add a new boolean attribute to the `KojiBuild` model that shows if the build is `scratch build` or not.
* [ ] Provide a migration script via alembic.
  * [ ] Fill the attribute for older data -- models connected to upstream projects are `scratch` and the builds connected to dist-git repositories are `non-scratch` ones.

### PR DESCRIPTION
**Pull request by majamassarini**
Migration has been tested on staging and production database

Fixes #1338

### CODE REVIEW

File: alembic/versions/46b37040cb19_add_koji_build_scratch_attribute.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    job_triggers = session.query(JobTriggerModel).all()
+    for job in job_triggers:
+        trigger = job.get_trigger_object()
+        for run in job.runs:
+            if run.koji_build:
+                run.koji_build.scratch = (
+                    False
+                    if "src.fedoraproject.org" in trigger.project.instance_url
+                    else True
+                )
```
Comment (lachmanfrantisek) [inline alembic/versions/46b37040cb19_add_koji_build_scratch_attribute.py:40]:
```suggestion
                run.koji_build.scratch = (
                    "src.fedoraproject.org" not in trigger.project.instance_url
                )
```

**Review by lachmanfrantisek — APPROVED** [review]
Nice!

**Review by TomasTomecek — APPROVED** [review]
nice!

**Review by lachmanfrantisek — APPROVED** [review]
Thanks!

**Comment by lachmanfrantisek** [comment]
I think you need to rebase to get the pre-commit fixed.

**Comment by majamassarini** [comment]
\recheck

**Comment by majamassarini** [comment]
recheck

**Pull request merged**